### DISCUSSION TITLE
Auth and worldserver dying when relogging.

### DISCUSSION THREAD

**Discussion by senkuma** [azerothcore/azerothcore-wotlk#9621] [General]
Hey again. I'm here with more trouble. The AC server is working perfectly, except when I try to login a second time the auth server just dies. I'm not getting anything in the auth log, but I got something about "Asynchronous connections on DatabasePool" before, now it's not showing anything.

So to sum up. Logging in works one time, when I try to login again with the same account or another I get stuck at "Authenticating" and then the auth dies. If I restart the auth server, login and choose realm, the world server dies. It works to login one time if both auth and world is restarted, then after the next login the error appears.

This is the latest crash log - [https://pastebin.com/BkmSpEjg](url)

I got the same problem on TrinityCore so it shouldn't be any wrong with the server per say. I can't think of what the problem could be.

**Comment by Eronox** [comment]
Seems to be something caused by boost. 

Try update your boost version to `1.71` or higher

Your crash happends here:
``` 
  _socket.async_read_some(boost::asio::buffer(_readBuffer.GetWritePointer(), _readBuffer.GetRemainingSpace()),
            std::bind(&Socket<T>::ReadHandlerInternal, this->shared_from_this(), std::placeholders::_1, std::placeholders::_2));
```
            
            
triggered by  `AsyncRead` in  ` AuthSession::CheckIpCallback` @ server/authserver/Server/AuthSession.cpp

**Reply by senkuma** [reply]
Yeah does look like that could be the problem, but I really can't use any other versions for some reason, I just get an error, no matter what version I try.

https://pastebin.com/iitXB3kJ

**Reply by Eronox** [reply]
Seems like it can't find the libraries, try set `-DBoost_USE_STATIC_LIBS=ON` and double check that these variables are set properly:

`BOOST_LIBRARYDIR `
 `BOOST_ROOT`

**Reply by senkuma** [reply]
The variables should be set properly, however I'm not sure where to find "-DBoost_USE_STATIC_LIBS=ON".

**Reply by senkuma** [reply]
I managed to get it working, I needed a specific boost for it to work. In my case "Boost 1.71.0 lib64-msvc-14.2".

So that's awesome, but I have one more problem. I can't chat in /say. Everything else works /p, /2, /g and so forth. But /say just doesn't show anything in chat window nor any chat bubbles. Any idea what it could be?

**Reply by Eronox** [reply]
Great :) 

`/say` is most likely just a chat window configurement mistake. Try hover over the chat window, specifically "general" tab, and go into the settings.

In there, see if it's even ticked 😄  
Another quick test would be to choose "reset windows" or whatever that option is called

**Reply by senkuma** [reply]
Yeah I have tried that all ready but it doesn't work :/

**Reply by Eronox** [reply]
> Yeah I have tried that all ready but it doesn't work :/

Try backup your WTF & cache folder, login, and see if it persists

**Reply by senkuma** [reply]
Yeah I did that before, I also tried redownloading WoW from another place. I even compiled a trinity server to see if it worked and it didn't. But it works when I login to another WoW private server without any problems.

**Reply by Eronox** [reply]
I've been asking other contributors if they had any idea and a very roughly idea we came up with was if it would be if it was your firewall causing.

Taking this into consideration:

A) no matter core,  but local hosted
B) connecting to any other server, it works

So try start your local server with firewall off just to verify, although its a veeery rough idea
